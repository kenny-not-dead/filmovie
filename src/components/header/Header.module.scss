.container {
	display: flex;
	align-items: center;
	justify-content: space-between;
	width: 100%;
	height: 70px;
	

	font-weight: 500;

	div {
		display: flex;
		gap: 40px;

		nav {
			display: flex;
			gap: 20px;
			margin: auto 0;
		}

		img {
			width: 150px;
		}
	}
}