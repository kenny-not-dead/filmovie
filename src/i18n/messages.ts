import { LOCALES } from './locales';

export const messages = {
	[LOCALES.ENGLISH]: {
		// header
		main: `Main`,
		films: 'Films',
		admin: 'Admin',
		auth: 'Sing up',


		// footer
		aboutus: 'About us',
			aboutcompani: 'About company',
			vacancy: 'vacancy',
			beta: 'Beta testing program',
			partners: 'Information for partners',
			advertising: "Advertising placement",
			agreement: 'User agreement',
			privacy: 'Privacy policy',
			compliance: 'Compliance',
		sections: 'Sections',
		support: 'Support service',
		supportp1: 'We are always ready to help you.',
		supportp2: 'Our operators are online 24/7',
		answer: 'Answers to questions',
		noadvertising: 'Watch movies, TV series and cartoons without advertising',


		//button
		message: 'Write in the chat',
		load: 'download',
		appstore: 'App store',
		google: 'Google play',
		smart: 'Smart TV',
		available: 'Available',
		watch: 'Watch in'
	},
	[LOCALES.RUSSIAN]: {
		// header
		main: `Главная`,
		films: 'Фильмы',
		admin: 'Админка',
		auth: 'Войти',

		// footer
		aboutus: 'О нас',
			aboutcompani: 'О компании',
			vacancy: 'Вакансии',
			beta: 'Программа бета-тестирования',
			partners: 'Информация для партнёров',
			advertising: "Размещение рекламы",
			agreement: 'Пользовательское соглашение',
			privacy: 'Политика конфиденциальности',
			compliance: 'Комплаенс',
		sections: 'Разделы',
		support: 'Служба поддержки',
		supportp1: 'Мы всегда готовы вам помочь.',
		supportp2: 'Наши операторы онлайн 24/7',
		answer: 'Ответы на вопросы',
		noadvertising:'Смотрите фильмы, сериалы и мультфильмы без рекламы',
		
		//button
		message: 'Написать в чате',
		load: 'Загрузить в',
		appstore: 'App store',
		google: 'Google play',
		smart: 'Smart TV',
		available: 'Доступно в',
		watch: 'Смотрите на'
	},
};

/*



*/

