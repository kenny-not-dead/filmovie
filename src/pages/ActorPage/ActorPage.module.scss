@import './../../styles/variables.scss';

.wrapper{
    display: grid;
    width: 700px;
    margin: 0 auto;
        p{
        color: $color-text-second;
        }
}



.infowrapper{
    display: flex;
    padding: 50px 0 30px;
    div {
        margin-left: 50px;
    }
    img{
        width: 150px;
        border-radius: 30px;
    }
}

.showall {
    margin-top: 15px;
    cursor: pointer;
    &:hover{
        color: $color-text ;
    }
}

.nav{
    display: flex;
    margin: 30px 0;
    font-size: 22px;
    color: $color-text-second ;
    gap: 10px;
    a{
        &:hover{
            color: $color-text ;
        }

    } 
}

.listlabel{
    margin: 30px 0 20px 0;

    display: flex;
    p{
        margin: auto 0 auto 15px;
    }
}

.listitem{
    display: flex;
    justify-content: space-between;
    padding: 10px 0;
    img{
        width: 100px
    };

}

.infolistwrapper{
    margin: auto auto auto 50px;
    p{
        padding: 2px 0;
    }
}

.buttonwrapper{
    margin: auto 0;
}

.bioWrapper{
    margin:  50px 0;
    p{
        margin-top: 20px;
    }
}