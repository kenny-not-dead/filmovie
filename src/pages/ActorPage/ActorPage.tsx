import { useState } from 'react';
import classes from './ActorPage.module.scss'
import Point from '../../components/UI/elem/Point';
import { divide } from 'lodash';
import { Button } from '../../components/UI/button/Button';
import { Title } from '../../components/title/Title';


let actors = [
	{name: 'Сергей Бурунов',
	 english: "Sergey Burunov",
	 text: `Сергей Бурунов – российский актер театра и кино, актер озвучивания и пародист. 
	 Получил известность по телешоу «Большая разница», после чего начал активно сниматься на телевидении и на большом экране. 
	 Бурунов снялся в таких сериалах, как «Краткий курс счастливой жизни», «Кухня» и «Полицейский с Рублевки», 
	 дилогии «Последний богатырь» и картине «Глубже!» (2020). Востребованный актер дубляжа - его голосом 
	 на русском языке «говорят» такие актеры, как Леонардо ДиКаприо, Джонни Депп, Джонатан Риз Майерс, 
	 Мэтт Дэймон и другие голливудские звезды.`,
	 film: [ 
		{
			title: 'Губка боб',
			url: 'https://thumbs.dfs.ivi.ru/storage2/contents/1/1/24e317e380f8a6bf033e25c3b47cc9.jpg/234x360/?q=85',
			score: '7,9',
			info: '1999-2021, США, Фентези',
			seasons: '13 сезонов',
			years: 2020
		},
		{
			title: 'Титаник',
			url: 'https://thumbs.dfs.ivi.ru/storage23/contents/8/5/d8d2fea60b6a57adb2d25a76499f58.jpg/234x360/?q=85',
			score: '8,6',
			info: '2012, Россия, Документальные',
			seasons: '1 сезон',
			years: 2022
		},
		{
			title: 'Друзья',
			url: 'https://thumbs.dfs.ivi.ru/storage8/contents/e/6/071e0668ff2d48216b7967891ca39f.jpg/234x360/?q=85',
			score: '9,1',
			info: '1994-2004, США, Комедийные',
			seasons: '10 сезонов',
			years: 2021
		},
	 ],
	 bio: `Сергей Александрович Бурунов родился в семье электротехника и медсестры 6 марта 1977 года в Москве. У Сергея есть старший брат, который стал переводчиком. В детстве Сергей и не мечтал становиться актером – он хотел летать и грезил о профессии военного летчика. Именно поэтому Сергей поступил в Качинское высшее авиационное училище летчиков им. А. Ф. Мясникова, где отучился три года. На службе Сергей впервые попробовал себя в комедийном жанре – в училище была местная команда КВН, и молодой студент с удовольствием принимал участие в пародийных постановках. В конце концов, Сергей понял, что далек от армии и бросил вуз. Оставшись на распутье, 20-летний Бурунов долго маялся, не зная, чем бы он хотел заниматься. Ответы Сергей нашел… в церкви, где пообщался со священником и получил от него номер телефона своего будущего педагога Михаила Петровича Симакова.

	 Поступить в знаменитое Щукинское училище Бурунову удалось только со второй попытки. Провалившись в первый год поступления, Сергей поступил в Эстрадно-цирковое училище имени Румянцева. Однако на будущий год 21-летнему Бурунову все же удалось стать студентом Щуки. После окончания театрального вуза 25-летний Бурунов четыре года играл на подмостках Театра Сатиры, а затем еще четыре года был голосом канала МГТРК «Мир», диктором анонсов радио «Россия». Прежде чем страна узнала актера Сергея Бурунова в лицо, сначала она полюбила его знаменитый голос. Бурунов много работает в дубляже популярных иностранных фильмов, мультиков и озвучивает компьютерные игры. Именно голосом Бурунова «говорят» звезды Голливуда – например, Леонардо ДиКаприо в фильмах «Кровавый алмаз», «Дорога перемен», «Начало», «Дж.Эдгар», или Джонни Депп в лентах «Мрачные тени», «Джонни Д.».
	 
	 Как актер Сергей дебютировал в 2003 году в сериале «Москва. Центральный округ», снявшись в роли Репейникова в молодости. В 2005 году на экраны вышла военная драма «Эшелон», в которой Бурунов прекрасно отыграл роль политрука, заслужив любовь и признание зрителей. Впрочем, настоящая известность пришла к Сергею в 2008 году, когда он стал постоянным актером рейтингового комедийного шоу на Первом канале «Большая разница». Там он заявил о себе как талантливый комик и пародист, сумев виртуозно изобразить массу знаменитостей. Параллельно с успехами в развлекательных программах пошла в гору карьера Бурунова и в телесериалах. В картинах «Отблески» (2009) и «Без права на ошибку» (2010) Бурунов исполнил уже главные роли. В 2011 году успех актера закрепился работой в сериале «Краткий курс счастливой жизни», где он исполнил роль брутального полицейского Тимура Валеева, с которым заводит долгоиграющий роман одна из главных героинь сериала. Бурунов появляется на экране далеко не сразу, но благодаря яркому образу и неординарным поступкам сумел запомниться, как едва ли не лучший мужской персонаж скандального проекта Валерии Гай Германики.
	 
	 В статусе ведущего комедийного актера страны Бурунов появился в роли флегматичного майора ФМС Ивановского в популярном комедийном сериале канала СТС «Кухня» (2012). Двумя годами позже Бурунов снялся сразу в нескольких полнометражных фильмах, самыми успешными из них стали «В спорте только девушки» и «Неформат». В том же году зрители Первого канала увидели Бурунова в телешоу «Повтори!» В 2016 году Сергей приступил к исполнению самой известной своей роли – подполковника Владимира Яковлева в комедийном сериале «Полицейский с Рублевки». Бурунов составил с другой ярчайшей звездой российского кино Александром Петровым прекрасный тандем. В отличие от дерзкого опера угрозыска Измайлова, сыгранного Петровым, персонаж Бурунова показан как добросовестный и душевный сыщик, которому по роду деятельности приходится вникать в проблемы не самых приятных, зато очень обеспеченных жителей страны. Образ Яковлева так полюбился зрителям, что сериал просуществовал на телевидении в течение пяти сезонов, а затем получил два полнометражных продолжения «Полицейский с Рублевки. Новогодний беспредел» (2018) и «Полицейский с Рублевки. Новогодний беспредел 2» (2019).
	 
	 2017 год становится пиком востребованности Бурунова, который становится настолько популярным, что от ролей просто нет отбоя. Сергей появился в фэнтезийной комедии «Последний богатырь», где сыграл Водяного. Несмотря на второстепенный характер роли и сложный грим, Бурунов запомнился зрителям наряду с главной звездой ленты Виктором Хориняком и Константином Лавроненко, воплотившим Кощея. Именно на счету Водяного самые удачные шутки фильма, Бурунов без преувеличения стал душой картины, ставшей коммерческим хитом. Еще один фильм 2017 года, в котором снялся Сергей – «Ну, здравствуй, Оксана Соколова!», где актеру досталась роль врача-фониатра. Также Бурунов принял участие в драме «Жизнь впереди» (2017), посвященной воссоединению бывших одноклассников. Наконец, в том же в 2017-м Сергей появился в научно-фантастическом фильме «Лачуга должника», посвященном экспериментам, увеличивающим продолжительность жизни. В комедийном сериале «Ивановы-Ивановы» Сергей сыграл крупного бизнесмена Антона Павловича Иванова, вынужденного в силу ряда обстоятельств скрывать свое дворянское происхождение.
	 
	 Бурунов продолжает активно сниматься на телевидение. В 2018 году к уже имеющимся проектам «Полицейский с Рублевки» и «Ивановы-Ивановы» добавился сериал «Содержанки», а годом позже – «Мылодрама». В 2020-м на экраны выходит нашумевшая мелодрама «Глубже!», становящаяся неожиданным триумфом проката. В этом фильме Бурунов по причине сильной загруженности исполняет небольшую роль телеведущего. Столь же фрагментарным получается участие Сергея в продолжении «Последнего богатыря» - ленте «Последний богатырь. Корень зла» (2021). Бурунов вернулся к роли Водяного, но появился буквально в паре сцен, став очень яркой, но эпизодической звездой успешной картины.
	 
	 Помимо успешной актерской карьеры, Бурунов много работает над записью музыкальных клипов. О личной жизни Сергей предпочитает не распространяться, но в настоящее время всенародно любимый комик не женат.`
	}

]




export function ActorPage() {

    const [number, setNumber] = useState(150)
	const [numberBio, setnumberBio] = useState(300)

	const setAll = (number: Number) => {
		if  (number == 150) {
			setNumber(1000000000000)
		} else if (number == 1000000000000) {
			setNumber(150)
		} else if(number == 300) {
		setnumberBio(1000000000000000)
        } else if (number == 1000000000000000) {
            setnumberBio(300)
        }
    } 

	return (
		<div className={classes.wrapper}>
			<div className={classes.infowrapper}>
				<img src="https://world-wells.ru/wp-content/uploads/0/f/5/0f564624972dad2115ea70d4b0bff5b0.jpeg" alt="Photo"  />
				<div>
					<h2>{actors[0].name}</h2>
					<p>{actors[0].english}</p>
				</div>
			</div>
			<p>{actors[0].text.slice(0, number)}{number == 150 ? '...' : ''} </p>
			<p className ={classes.showall} onClick={() => setAll(number)}>{number == 150 ? 'Развернуть' : 'Свернуть'}</p>
			<div className={classes.nav}>
				<a href="#film">{actors[0].film.length} фильма</a>
				<Point/>
				<a href="#bio">Биография</a>
			</div>
			<div id='film'>
				<div className={classes.listlabel} >
					<Title arrow={false} title={'Полная фильмография'} />
					<p>{actors[0].film.length} фильма</p>
				</div>

				{actors[0].film.map(item => 
					<div className={classes.listitem}>
						<img src={item.url} alt="Poster" />
						<div className={classes.infolistwrapper}>
							<h3>{item.title}</h3>
							<p>{item.years}</p>
							<p>Рейтинг Кинопоиск: {item.score}</p>
						</div>
						<div className={classes.buttonwrapper}>
							<Button id ='Смотреть'/>
						</div>
						
					</div>
				)}
				<div className={classes.bioWrapper} id='bio'>
					<Title arrow={false} title={'Биография'} />
					<p>
						<p>{actors[0].bio.slice(0, numberBio)}{numberBio == 300 ? '...' : ''} </p>
						<p className ={classes.showall} onClick={() => setAll(numberBio)}>{numberBio == 300 ? 'Развернуть' : 'Свернуть'}</p>
					</p>
				</div>
		
			</div>
		</div>
	);
}