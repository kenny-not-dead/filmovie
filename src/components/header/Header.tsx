import { FormattedMessage } from "react-intl";
import { NavLink } from "react-router-dom";
import classes from "./Header.module.scss";
import logo from "./../../static/svgs/logo.svg";
import { LOCALES } from "../../i18n/locales";

interface SubfilterType {
  currentLocale: any;
  hangeChange: () => {};
}

export function Header(props: SubfilterType) {
  const languages = [
    { name: "English", code: LOCALES.ENGLISH },
    { name: "Русский", code: LOCALES.RUSSIAN },
  ];

  return (
    <header className={classes.container}>
      <div>
        <img src={logo} alt="Kinogramm" />
        <nav>
          <NavLink to={"/"}>
            <FormattedMessage id="main" />
          </NavLink>
          <NavLink to={"/catalog"}>
            <FormattedMessage id="films" />
          </NavLink>
          <NavLink to={"/admin"}>
            <FormattedMessage id="admin" />
          </NavLink>
        </nav>
      </div>
      <NavLink to={"/auth"}>
        <FormattedMessage id="auth" />
      </NavLink>
      <div className={classes.switcher}>
        <select onChange={props.hangeChange} value={props.currentLocale}>
          {languages.map(({ name, code }) => (
            <option key={code} value={code}>
              {name}
            </option>
          ))}
        </select>
      </div>
    </header>
  );
}
