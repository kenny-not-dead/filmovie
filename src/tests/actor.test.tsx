import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { MemoryRouter } from "react-router-dom";
import { Header } from "../components/header/Header";
import { LOCALES } from "../i18n/locales";
import { IntlProvider } from "react-intl";
import { messages } from "../i18n/messages";
import { ActorPage } from "../pages/ActorPage/ActorPage";

describe("render components", () => {
  it("should create main page", () => {
    const actorPage = render(
      <MemoryRouter>
        <IntlProvider
          messages={messages[LOCALES.RUSSIAN]}
          locale={LOCALES.RUSSIAN}
        >
          <ActorPage />
        </IntlProvider>
      </MemoryRouter>
    );
    expect(actorPage).toMatchSnapshot();
  });

  test("render header", () => {
    render(
      <MemoryRouter>
        <IntlProvider
          messages={messages[LOCALES.RUSSIAN]}
          locale={LOCALES.RUSSIAN}
        >
          <Header
            currentLocale={LOCALES.RUSSIAN}
            hangeChange={() => {
              return {};
            }}
          />
        </IntlProvider>
      </MemoryRouter>
    );
    expect(screen.getByText(/главная/i)).toBeInTheDocument();
    expect(screen.getByText(/фильмы/i)).toBeInTheDocument();
    expect(screen.getByText(/админка/i)).toBeInTheDocument();
    expect(screen.getByText(/войти/i)).toBeInTheDocument();
  });

  test("render actor info", () => {
    const name = "Сергей Бурунов";
    const nameEnglish = "Sergey Burunov";

    render(
      <MemoryRouter>
        <IntlProvider
          messages={messages[LOCALES.RUSSIAN]}
          locale={LOCALES.RUSSIAN}
        >
          <ActorPage />
        </IntlProvider>
      </MemoryRouter>
    );
    expect(screen.getAllByText(name).length).toBe(14);
    expect(screen.getAllByText(nameEnglish).length).toBe(14);
  });

  test("render buttons", () => {
    render(
      <MemoryRouter>
        <IntlProvider
          messages={messages[LOCALES.RUSSIAN]}
          locale={LOCALES.RUSSIAN}
        >
          <ActorPage />
        </IntlProvider>
      </MemoryRouter>
    );
    expect(screen.getByText(/Смотреть/i)).toBeInTheDocument();
  });
});
