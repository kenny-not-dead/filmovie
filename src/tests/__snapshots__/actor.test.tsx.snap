// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render components should create main page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="test"
      >
        <img
          alt=""
          class="bgimg"
          src="https://thumbs.dfs.ivi.ru/storage23/contents/2/b/0c693b91b7b3361068d637b42e3ed0.jpg/1920x1080/?q=85"
        />
      </div>
      <div
        class="wrapper"
      >
        <div
          class="infowrapper"
        >
          <img
            alt="Photo"
            src="https://world-wells.ru/wp-content/uploads/0/f/5/0f564624972dad2115ea70d4b0bff5b0.jpeg"
          />
          <div>
            <h2>
              Сергей Бурунов
            </h2>
            <p>
              Sergey Burunov
            </p>
          </div>
        </div>
        <div
          class="wraper"
        >
          <p>
            Сергей Бурунов – российский актер театра и кино, актер озвучивания и пародист. 
	 Получил известность по телешоу «Большая разница», после чего начал а
            ...
             
          </p>
          <p
            class="showall"
          >
            Развернуть
          </p>
        </div>
        <div
          class="nav"
        >
          <a
            href="#film"
          >
            3
             фильма
          </a>
          <div
            class="point"
          />
          <a
            href="#bio"
          >
            Биография
          </a>
        </div>
        <div
          id="film"
        >
          <div
            class="listlabel"
          >
            <div
              class="container"
              data-testid="title-container"
            >
              <h2
                class="title"
              >
                Полная фильмография
              </h2>
            </div>
            <p>
              3
               фильма
            </p>
          </div>
          <div
            class="listitem"
          >
            <img
              alt="Poster"
              src="https://thumbs.dfs.ivi.ru/storage2/contents/1/1/24e317e380f8a6bf033e25c3b47cc9.jpg/234x360/?q=85"
            />
            <div
              class="infolistwrapper"
            >
              <h3>
                Губка боб
              </h3>
              <p>
                2020
              </p>
              <p>
                Рейтинг Кинопоиск: 
                7,9
              </p>
            </div>
            <div
              class="buttonwrapper"
            >
              <button
                class="btn"
              >
                <p>
                  Смотреть
                </p>
              </button>
            </div>
          </div>
          <div
            class="listitem"
          >
            <img
              alt="Poster"
              src="https://thumbs.dfs.ivi.ru/storage23/contents/8/5/d8d2fea60b6a57adb2d25a76499f58.jpg/234x360/?q=85"
            />
            <div
              class="infolistwrapper"
            >
              <h3>
                Титаник
              </h3>
              <p>
                2022
              </p>
              <p>
                Рейтинг Кинопоиск: 
                8,6
              </p>
            </div>
            <div
              class="buttonwrapper"
            >
              <button
                class="btn"
              >
                <p>
                  Смотреть
                </p>
              </button>
            </div>
          </div>
          <div
            class="listitem"
          >
            <img
              alt="Poster"
              src="https://thumbs.dfs.ivi.ru/storage8/contents/e/6/071e0668ff2d48216b7967891ca39f.jpg/234x360/?q=85"
            />
            <div
              class="infolistwrapper"
            >
              <h3>
                Друзья
              </h3>
              <p>
                2021
              </p>
              <p>
                Рейтинг Кинопоиск: 
                9,1
              </p>
            </div>
            <div
              class="buttonwrapper"
            >
              <button
                class="btn"
              >
                <p>
                  Смотреть
                </p>
              </button>
            </div>
          </div>
          <div
            class="bioWrapper"
            id="bio"
          >
            <div
              class="container"
              data-testid="title-container"
            >
              <h2
                class="title"
              >
                Биография
              </h2>
            </div>
            <div
              class="wraper"
            >
              <p>
                Сергей Александрович Бурунов родился в семье электротехника и медсестры 6 марта 1977 года в Москве. У Сергея есть старший брат, который стал переводчиком. В детстве Сергей и не мечтал становиться актером – он хотел летать и грезил о профессии военного летчика. Именно поэтому Сергей поступил в Качинс
                ...
                 
              </p>
              <p
                class="showall"
              >
                Развернуть
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="test"
    >
      <img
        alt=""
        class="bgimg"
        src="https://thumbs.dfs.ivi.ru/storage23/contents/2/b/0c693b91b7b3361068d637b42e3ed0.jpg/1920x1080/?q=85"
      />
    </div>
    <div
      class="wrapper"
    >
      <div
        class="infowrapper"
      >
        <img
          alt="Photo"
          src="https://world-wells.ru/wp-content/uploads/0/f/5/0f564624972dad2115ea70d4b0bff5b0.jpeg"
        />
        <div>
          <h2>
            Сергей Бурунов
          </h2>
          <p>
            Sergey Burunov
          </p>
        </div>
      </div>
      <div
        class="wraper"
      >
        <p>
          Сергей Бурунов – российский актер театра и кино, актер озвучивания и пародист. 
	 Получил известность по телешоу «Большая разница», после чего начал а
          ...
           
        </p>
        <p
          class="showall"
        >
          Развернуть
        </p>
      </div>
      <div
        class="nav"
      >
        <a
          href="#film"
        >
          3
           фильма
        </a>
        <div
          class="point"
        />
        <a
          href="#bio"
        >
          Биография
        </a>
      </div>
      <div
        id="film"
      >
        <div
          class="listlabel"
        >
          <div
            class="container"
            data-testid="title-container"
          >
            <h2
              class="title"
            >
              Полная фильмография
            </h2>
          </div>
          <p>
            3
             фильма
          </p>
        </div>
        <div
          class="listitem"
        >
          <img
            alt="Poster"
            src="https://thumbs.dfs.ivi.ru/storage2/contents/1/1/24e317e380f8a6bf033e25c3b47cc9.jpg/234x360/?q=85"
          />
          <div
            class="infolistwrapper"
          >
            <h3>
              Губка боб
            </h3>
            <p>
              2020
            </p>
            <p>
              Рейтинг Кинопоиск: 
              7,9
            </p>
          </div>
          <div
            class="buttonwrapper"
          >
            <button
              class="btn"
            >
              <p>
                Смотреть
              </p>
            </button>
          </div>
        </div>
        <div
          class="listitem"
        >
          <img
            alt="Poster"
            src="https://thumbs.dfs.ivi.ru/storage23/contents/8/5/d8d2fea60b6a57adb2d25a76499f58.jpg/234x360/?q=85"
          />
          <div
            class="infolistwrapper"
          >
            <h3>
              Титаник
            </h3>
            <p>
              2022
            </p>
            <p>
              Рейтинг Кинопоиск: 
              8,6
            </p>
          </div>
          <div
            class="buttonwrapper"
          >
            <button
              class="btn"
            >
              <p>
                Смотреть
              </p>
            </button>
          </div>
        </div>
        <div
          class="listitem"
        >
          <img
            alt="Poster"
            src="https://thumbs.dfs.ivi.ru/storage8/contents/e/6/071e0668ff2d48216b7967891ca39f.jpg/234x360/?q=85"
          />
          <div
            class="infolistwrapper"
          >
            <h3>
              Друзья
            </h3>
            <p>
              2021
            </p>
            <p>
              Рейтинг Кинопоиск: 
              9,1
            </p>
          </div>
          <div
            class="buttonwrapper"
          >
            <button
              class="btn"
            >
              <p>
                Смотреть
              </p>
            </button>
          </div>
        </div>
        <div
          class="bioWrapper"
          id="bio"
        >
          <div
            class="container"
            data-testid="title-container"
          >
            <h2
              class="title"
            >
              Биография
            </h2>
          </div>
          <div
            class="wraper"
          >
            <p>
              Сергей Александрович Бурунов родился в семье электротехника и медсестры 6 марта 1977 года в Москве. У Сергея есть старший брат, который стал переводчиком. В детстве Сергей и не мечтал становиться актером – он хотел летать и грезил о профессии военного летчика. Именно поэтому Сергей поступил в Качинс
              ...
               
            </p>
            <p
              class="showall"
            >
              Развернуть
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
